x/market/handler/endblock.go

```diff
@@ -52,7 +52,7 @@ func transferFundsForActiveLeases(ctx sdk.Context, keepers Keepers) error {
 	return nil
 }
 
-var errNoBids error = errors.New("no bids to pick winner from")
+var ErrNoBids error = errors.New("no bids to pick winner from")
```

Comment:
the `error` isn't needed here.  I think that this wasn't exported because it's not used anywhere, would prefer to keep it that way.

Reply:
This was causing a failing test, and i exported it to make the `error.Is` call pass properly.